src/transformers/mutableClone.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  DDTransformerProps,
+  DDTransformerState,
+} from "../transformer-components/DataDrivenTransformer";
+import { getContextAndDataSet } from "../utils/codapPhone";
+import { applyNewDataSet, readableName } from "../transformer-components/util";
+import { uncheckedCopy } from "./copy";
+
+export async function mutableCloneOverride(
+  { setErrMsg }: DDTransformerProps,
+  { context1: inputDataCtxt }: DDTransformerState
+): Promise<void> {
+  setErrMsg(null);
+
+  if (inputDataCtxt === null) {
+    setErrMsg("Please choose a valid dataset to transform");
+    return;
+  }
+
+  const { context, dataset } = await getContextAndDataSet(inputDataCtxt);
+  const ctxtName = readableName(context);
+
+  applyNewDataSet(
+    uncheckedCopy(dataset),
+    `Clone of ${ctxtName}`,
+    `A copy of the ${ctxtName} dataset.`
```

Comment:
I think the dataset description here should indicate the special properties of this transformer: that it is a copy, it is mutable, and it won't update when the input does.